### PR TITLE
InstData Import: make the batch importer [formerly known as Manager, now called launch] a genome process.

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

**Comment by davidlmorton** [comment]
I didn't have time to review it all, but the process related stuff looks good to me.

**Comment by ebelter** [comment]
CI, test this.

**Comment by ebelter** [comment]
@davidlmorton Thanks you.

File: lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/Inputs.pm [hunk 1]
```diff
@@ -9,74 +9,60 @@ use Genome::InstrumentData::Command::Import::WorkFlow::SourceFiles;
...
-    has_transient => {
-        incoming_params => { is => 'HASH', },
-        instrument_data_properties => { is => 'HASH', },
+    has => {
+        analysis_project => { is => 'Genome::Config::AnalysisProject', },
+        library => { is => 'Genome::Library', },
+        instrument_data_properties => { is => 'HASH', default_value => {}, },
         source_files => { is => 'Genome::InstrumentData::Command::Import::WorkFlow::SourceFiles', },
+    },
+    has_transient => {
```
Comment (tmooney) [inline lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/Inputs.pm]: I don't think its necessary for these `via`/`to` properties to be `transient` if the properties they are `via` are not.

**Comment by ebelter** [comment]
@tmooney I will update in another PR - I have a few small additions to make.

**Pull request merged**